src/api/admin.ts

```diff
@@ -217,40 +218,61 @@ export const createVenue_v2 = async (
   input: WithWorldId<Omit<VenueInput_v2, "id">>,
   userId: UserId
 ) => {
+  const inputOptions = {
+    ...input,
+    showShoutouts: input.showShoutouts ?? DEFAULT_SHOW_SHOUTOUTS,
+    showReactions: input.showReactions ?? DEFAULT_SHOW_REACTIONS,
+    rooms: [],
+  };
   const firestoreVenueInput = await createFirestoreVenueInputWithoutId_v2(
-    {
-      ...input,
-      showShoutouts: input.showShoutouts ?? DEFAULT_SHOW_SHOUTOUTS,
-      showReactions: input.showReactions ?? DEFAULT_SHOW_REACTIONS,
-      rooms: [],
-    },
+    inputOptions,
     userId
+  ).catch(
+    createErrorRethrow({
+      where: "createVenue_v2",
```

Comment:
[NB] Would be good to create variables to avoid typos, unless this is TS and it will cry with a compile error.

Reply:
It is purely informational. Later on it may be added to a Bugsnag context (easy place for the Bugsnag would be in the parent error)